client/blocks/product-purchase-features/product-purchase-features-list/index.jsx

```diff
@@ -77,6 +78,10 @@ export default class ProductPurchaseFeaturesList extends Component {
 				selectedSite={ selectedSite }
 				key="customizeThemeFeature"
 			/>,
+			<LiveCourses
+				isBusinessPlan
```

Comment:
Is this prop necessary? It seems to be used in `AdvertisingRemoved` to alter the feature description for Premium and Business, but doesn't look like it's used inside the `LiveCourses` component